packages/luigis-box/src/Resources/config/services.yaml

```diff
@@ -7,11 +7,12 @@ services:
     Shopsys\LuigisBoxBundle\Component\LuigisBox\LuigisBoxClient:
         arguments:
             $luigisBoxApiUrl: '%env(string:LUIGIS_BOX_API_URL)%'
+            $trackerIdsByDomainIds: '%env(LUIGIS_BOX_TRACKER_IDS_BY_DOMAIN_IDS)%'
 
     Shopsys\LuigisBoxBundle\Component\LuigisBox\Filter\ProductFilterToLuigisBoxFilterMapper: ~
 
     Shopsys\LuigisBoxBundle\Model\Product\ProductSearchResultsProvider:
         arguments:
-            $enabledDomainIds: '%luigisBox.enabled_domain_ids%'
+            $enabledDomainIds: '%env(LUIGIS_BOX_ENABLED_DOMAIN_IDS)%'
```

Comment:
is a possible to use JSON env preprocessor? https://symfony.com/doc/current/configuration/env_var_processors.html